src/node-diff3.test.ts

```diff
@@ -1,92 +1,164 @@
-import { diff3MergeIndices, diffIndices, LCS } from './node-diff3';
+import { diff3MergeIndices, diffIndices, LCS, makeRange } from './node-diff3';
 
 describe('diff3MergeIndices', () => {
   it('works with one-sided change', () => {
     expect(diff3MergeIndices('', 'hello', '')).toEqual([
-      [-1, 0, 0, 0, 5, 0, 0],
```

Comment:
All of these tests were wrong before -- the names of the tests were assuming an O/L/R ordering of arguments, but `diff3MergeIndices` had a L/O/R ordering before this PR. So the hardcoded `expect` results were incorrect.